compose.yml

```diff
@@ -119,8 +122,10 @@ services:
     depends_on:
       db:
         condition: service_healthy
-    #      bff-migration:
-    #        condition: service_completed_successfully
+      redis:
+        condition: service_healthy
```

Comment:
håper jeg ikke går i beina for noen her nå. La inn litt endringer i compose også